sdk/bin/download-protoc.js

```diff
@@ -7,9 +7,10 @@ const path = require('path');
 const fs = require('fs');
 const rimraf = require('rimraf');
 
+// https://github.com/protocolbuffers/protobuf/releases/download/v3.25.2/protoc-3.25.2-linux-x86_64.zip
 const downloadUrlPrefix =
   'https://github.com/protocolbuffers/protobuf/releases/download/v';
-const protocVersion = '3.20.1';
+const protocVersion = '25.2';
```

Comment:
Was this needed in the end? Should we keep as it was to not introduce other changes in the same PR (might be a risk that the typescript protobuf library would need to be bumped along with protoc, not sure)?

Reply:
Nope, reverting